src/components/Page/tests/Page.test.tsx

```diff
@@ -42,6 +42,18 @@ describe('<Page />', () => {
     };
   }
 
+  describe('forceRender renders children in page', () => {
+    const {
+      createSpy: titleBarCreateSpy,
+      restore: restoreTitleBarCreateMock,
+    } = mockTitleBarCreate();
+    const {page} = mountWithAppBridge(<Page {...mockProps} forceRender />);
```

Comment:
instead of using just `{...mockProps}`, I would do `{...mockProps} title="new title"`

this ensure even if the default props change, this test won't fail.